### DISCUSSION TITLE
Per-app language selection for TeamTalk on Android

### DISCUSSION THREAD

**Discussion by beqabeqa473** [BearWare/TeamTalk5#2022] [Ideas]
### Discussed in https://github.com/BearWare/TeamTalk5/discussions/1623

<div type='discussions-op-text'>

<sup>Originally posted by **Nardol** December  8, 2022</sup>
Hello,

Since Android 13, it is possible to select language per app in the phone settings->System->Languages.

Could it be possible to have it implemented in TeamTalk for Android?

There is documentation here: https://developer.android.com/guide/topics/resources/app-languages</div>

**Comment by bear101** [comment]
@beqabeqa473 I would think so. I don't see why it wouldn't be possible.

**Comment by beqabeqa473** [comment]
It is definitely possible. We just need to find a way of automatically create xml to not fill new languages by hand.

**Comment by beqabeqa473** [comment]
will be fixed in #2035

**Comment by beqabeqa473** [comment]
merged to master
